tests/e2e/utils/DriverHelper.ts

```diff
@@ -273,12 +279,23 @@ export class DriverHelper {
                 await element.click();
                 return;
             } catch (err) {
-                if (err instanceof error.StaleElementReferenceError) {
-                    Logger.debug(`DriverHelper.waitAndClik - ${elementLocator} - StaleElementReferenceError - ${err}`);
+                if (err instanceof error.StaleElementReferenceError || (err instanceof error.ElementClickInterceptedError && i !== attempts - 1)) {
+                    Logger.debug(`DriverHelper.waitAndClick - ${elementLocator} - ${err}`);
                     await this.wait(polling);
                     continue;
                 }
 
+                if (err instanceof error.ElementClickInterceptedError && i === attempts - 1) {
+                    Logger.debug(`DriverHelper.waitAndClick - Element is not clickable, try to perform pointer click`);
+                    await this.getAction()
```

Comment:
Comment with explanation of which problem this code is solving and why `await element.click();` is not enough, would improve readability of the method significantly.